### PR TITLE
Clean up demo styles

### PR DESCRIPTION
**Pull request by corykon**
Inline styles ripped out, some minor things in markup/demo components cleaned up. Styled sort of like someone threw the catalyst style guide on top of angular material's doc site. (Make tweaks as you so please, @andrew-frueh.)

Also- the Left side navigation is styled like tabs, but not actually using tabs component. As soon as they can handle routerLink, anyone feel free to install them instead.

@joeskeen I hope I'm not making your life a nightmare with all these changes. Ping me if you need help with the merge

### CODE REVIEW

**Comment by corykon** [comment]
> The title bar for each component in its content area looks a little like our subheader, but it's not a big deal. I suppose we actually could use the subheader for that once I get it written.

@andrew-frueh That's what I was going for. Feel free to swap that component in when its built, or do something totally different if you'd like.

File: src/app/demo/side-nav/side-nav.component.html [hunk 1]
```diff
@@ -1,41 +1,41 @@
-<div class="side-nav">
+<div class="side-nav" routerLinkActive="active">
```
Comment (benjanderson) [inline src/app/demo/side-nav/side-nav.component.html]: This looks amazing, but its a tragedy you couldn't use the sidebar component

File: src/app/lib/drawer/drawer-demo/drawer-demo.component.html [hunk 2]
```diff
@@ -16,7 +20,7 @@ <h4 style="font-weight: 300; margin-top: 0">You can have both a left and right d
         </hc-drawer-container>
     </div>
 
-    <h4 style="font-weight: 300; margin-top: 0">You can have the drawer push or overlay your content</h4>
```
Comment (benjanderson) [inline src/app/lib/drawer/drawer-demo/drawer-demo.component.html]: great work cleaning out inline styles

File: src/app/lib/button/button-demo/button-demo.component.html [hunk 3]
```diff
@@ -28,113 +28,92 @@ <h5>
...
-        <div class="codeToggle">
-            <span class="toggleOptionOn" (click)="viewToggle('html')">HTML</span>
-            <span class="toggleOptionOff" (click)="viewToggle('ts')">TypeScript</span>
-        </div>
-        <pre>
-            <code>
+    <hr>
+
+    <h3>Angular Component</h3>
+    <hc-tab-set direction="horizontal">
```
Comment (benjanderson) [inline src/app/lib/button/button-demo/button-demo.component.html]: love the use of <hc-tab> :)

**Pull request merged**